specs/array_intersection_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require_relative '../lib/array_intersection'
+
+describe "array intersection" do
+  describe "basic tests" do
+    describe "test 1" do
```

Comment:
Same as the other repos, we should probably remove these leaf-most `describe` blocks.

Reply:
done